curriculum/README.md

```diff
@@ -0,0 +1,12 @@
+Curriculum
+==========
+
+| Specialization                                                  | Units      | Lead |
+|-----------------------------------------------------------------|------------|------|
+| [Bootcamp](0-bootcamp-md)                                       | SLUs 1-16  | TBD  |
+| [(Binary) Classification](1-binary-classification.md)           | SLUs 17-19 | TBD  |
+| [Time-Series](2-time-series.md)                                 | BLUs 1-3   | TBD  |
```

Comment:
That's a great question. Might make sense to change the order since the wrangling stuff is useful to the rest of the academy. Plus timeseries can include quite a bit of pandas-monium so it might make sense to give them another month to get more comfortable with it.

Reply:
I think this is a great idea, actually. Let's do it.

Reply:
Changing it after closing the bootcamp, if that's ok.